flang/test/Lower/array.f90

```diff
@@ -76,6 +76,26 @@ subroutine s(i,j,k,ii,jj,kk,a1,a2,a3,a4,a5,a6,a7)
   
 end subroutine s
 
+! CHECK: %[[VAL_1:.*]] = constant 1.000000e+00 : f32
```

Comment:
Could you add a CHECK-LABEL with the common block `fir.global  someName` here. Now that these checks are placed before the block data code, they are a bit weird to read out of any context.